src/System.Console/tests/TermInfo.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class TermInfo
+{
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        string[] locations = GetFieldValueOnObject<string[]>("_terminfoLocations", null, Type.GetType("ConsolePal.TermInfo.Database"));
+        foreach (string location in locations)
+        {
+            if (!Directory.Exists(location))
+                continue;
+
+            foreach (string term in Directory.EnumerateFiles(location, "*", SearchOption.AllDirectories))
+            {
+                if (term.ToUpper().Contains("README")) continue;
+                object info = CreateTermColorInfo(ReadTermInfoDatabase(Path.GetFileName(term)));
+
+                if (!string.IsNullOrEmpty(GetForegroundFormat(info)))
+                {
+                    Assert.NotEmpty(EvaluateParameterizedStrings(GetForegroundFormat(info), 0 /* irrelevant, just an integer to put into the formatting*/));
+                }
+
+                if (!string.IsNullOrEmpty(GetBackgroundFormat(info)))
+                {
+                    Assert.NotEmpty(EvaluateParameterizedStrings(GetBackgroundFormat(info), 0 /* irrelevant, just an integer to put into the formatting*/));
+                }
+            }
+        }
+    }
+
+    [Theory]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    [InlineData("xterm-256color", "\u001B\u005B\u00330m", "\u001B\u005B\u00340m", 0)]
+    [InlineData("xterm-256color", "\u001B\u005B\u00331m", "\u001B\u005B\u00341m", 1)]
+    [InlineData("xterm-256color", "\u001B\u005B\u00338m", "\u001B\u005B\u00348m", 8)]
```

Comment:
How did you arrive at the output strings for the new 8-15 versions?  They don't look right to me (but I haven't tried them).